src/adr/Program.cs

```diff
@@ -10,13 +14,20 @@ private static void Main(string[] args)
         {
             var app = new CommandLineApplication();
             app.Name = "adr-cli";
-            app.Description = "";
+            app.Description = "A Windows equivalent of adr-tools (https://github.com/npryce/adr-tools)";
```

Comment:
```suggestion
            app.Description = "A .Net equivalent of adr-tools (https://github.com/npryce/adr-tools)";
```

Reply:
I copied it from the repo description... ;-)